2022-10-11 03:43:44,997 INFO    Thread-8  :20914 [wandb_setup.py:_flush():75] Loading settings from /root/.config/wandb/settings
2022-10-11 03:43:44,997 INFO    Thread-8  :20914 [wandb_setup.py:_flush():75] Loading settings from wandb/settings
2022-10-11 03:43:44,997 INFO    Thread-8  :20914 [wandb_setup.py:_flush():75] Loading settings from environment variables: {'project': 'meme_experiments', 'entity': 'vietnguyen', 'root_dir': '/content/SSLMemes', 'run_id': 'rva7lx4a', 'sweep_param_path': '/content/SSLMemes/wandb/sweep-gz7p8ksn/config-rva7lx4a.yaml', 'sweep_id': 'gz7p8ksn'}
2022-10-11 03:43:44,998 INFO    Thread-8  :20914 [wandb_setup.py:_flush():75] Inferring run settings from compute environment: {'program_relpath': 'src/train.py', 'program': '/content/SSLMemes/src/train.py'}
2022-10-11 03:43:44,998 INFO    Thread-8  :20914 [wandb_init.py:_log_setup():405] Logging user logs to /content/SSLMemes/wandb/run-20221011_034344-rva7lx4a/logs/debug.log
2022-10-11 03:43:44,998 INFO    Thread-8  :20914 [wandb_init.py:_log_setup():406] Logging internal logs to /content/SSLMemes/wandb/run-20221011_034344-rva7lx4a/logs/debug-internal.log
2022-10-11 03:43:44,998 INFO    Thread-8  :20914 [wandb_init.py:init():439] calling init triggers
2022-10-11 03:43:44,998 INFO    Thread-8  :20914 [wandb_init.py:init():443] wandb.init called with sweep_config: {'batch_size': 32, 'dropout': 0, 'epochs': 1, 'hidden_size': 512, 'learning_rate': 0.050432488799879195, 'optimizer': 'sgd'}
config: {}
2022-10-11 03:43:44,998 INFO    Thread-8  :20914 [wandb_init.py:init():492] starting backend
2022-10-11 03:43:44,998 INFO    Thread-8  :20914 [backend.py:_multiprocessing_setup():101] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2022-10-11 03:43:45,002 INFO    Thread-8  :20914 [backend.py:ensure_launched():219] starting backend process...
2022-10-11 03:43:45,041 INFO    Thread-8  :20914 [backend.py:ensure_launched():225] started backend process with pid: 21679
2022-10-11 03:43:45,043 INFO    Thread-8  :20914 [wandb_init.py:init():501] backend started and connected
2022-10-11 03:43:45,052 INFO    Thread-8  :20914 [wandb_run.py:_config_callback():992] config_cb None None {'batch_size': 32, 'dropout': 0, 'epochs': 1, 'hidden_size': 512, 'learning_rate': 0.050432488799879195, 'optimizer': 'sgd'}
2022-10-11 03:43:45,059 INFO    Thread-8  :20914 [wandb_init.py:init():565] updated telemetry
2022-10-11 03:43:45,152 INFO    Thread-8  :20914 [wandb_init.py:init():596] communicating run to backend with 30 second timeout
2022-10-11 03:43:46,097 INFO    Thread-8  :20914 [wandb_run.py:_on_init():1759] communicating current version
2022-10-11 03:43:46,142 INFO    Thread-8  :20914 [wandb_run.py:_on_init():1763] got version response upgrade_message: "wandb version 0.13.4 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2022-10-11 03:43:46,142 INFO    Thread-8  :20914 [wandb_init.py:init():625] starting run threads in backend
2022-10-11 03:43:49,694 INFO    Thread-8  :20914 [wandb_run.py:_console_start():1733] atexit reg
2022-10-11 03:43:49,695 INFO    Thread-8  :20914 [wandb_run.py:_redirect():1606] redirect: SettingsConsole.REDIRECT
2022-10-11 03:43:49,695 INFO    Thread-8  :20914 [wandb_run.py:_redirect():1611] Redirecting console.
2022-10-11 03:43:49,695 ERROR   Thread-8  :20914 [wandb_run.py:_redirect():1670] Failed to redirect.
Traceback (most recent call last):
  File "/usr/local/lib/python3.7/dist-packages/wandb/sdk/wandb_run.py", line 1663, in _redirect
    out_redir.install()
  File "/usr/local/lib/python3.7/dist-packages/wandb/sdk/lib/redirect.py", line 709, in install
    _WSCH.add_fd(self._pipe_read_fd)
  File "/usr/local/lib/python3.7/dist-packages/wandb/sdk/lib/redirect.py", line 659, in add_fd
    self._register()
  File "/usr/local/lib/python3.7/dist-packages/wandb/sdk/lib/redirect.py", line 644, in _register
    old_handler = signal.signal(signal.SIGWINCH, lambda *_: None)
  File "/usr/lib/python3.7/signal.py", line 47, in signal
    handler = _signal.signal(_enum_to_int(signalnum), _enum_to_int(handler))
ValueError: signal only works in main thread
2022-10-11 03:43:49,696 INFO    Thread-8  :20914 [wandb_init.py:init():664] run started, returning control to user process
2022-10-11 03:43:49,696 INFO    Thread-8  :20914 [wandb_run.py:_config_callback():992] config_cb None None {'img_feature_path': 'data/features/visualbert/', 'train_csv_path': 'data/splits/random/memotion_train.csv', 'val_csv_path': 'data/splits/random/memotion_val.csv', 'model_type': 'visualbert', 'model_path': 'uclanlp/visualbert-nlvr2-coco-pre', 'eval_step': 100, 'amp': True, 'output_dir': './tmp', 'checkpoint_step': 100, 'random_seed': 42, 'resume_training': False, 'semi_supervised': False}
2022-10-11 03:43:50,078 INFO    Thread-8  :20914 [wandb_run.py:_config_callback():992] config_cb None None {'output_hidden': 768}
2022-10-11 03:45:29,544 INFO    Thread-8  :20914 [wandb_run.py:finish():1539] finishing run vietnguyen/meme_experiments/rva7lx4a
2022-10-11 03:45:29,545 INFO    Thread-8  :20914 [wandb_run.py:_atexit_cleanup():1702] got exitcode: 0
2022-10-11 03:45:29,547 INFO    Thread-8  :20914 [wandb_run.py:_restore():1674] restore
2022-10-11 03:45:29,551 INFO    Thread-8  :20914 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 699
  total_bytes: 699
}

2022-10-11 03:45:29,778 INFO    Thread-8  :20914 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 699
  total_bytes: 699
}

2022-10-11 03:45:30,154 INFO    Thread-8  :20914 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 699
  total_bytes: 10618
}

2022-10-11 03:45:30,256 INFO    Thread-8  :20914 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 699
  total_bytes: 10618
}

2022-10-11 03:45:30,358 INFO    Thread-8  :20914 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 10618
  total_bytes: 10618
}

2022-10-11 03:45:30,460 INFO    Thread-8  :20914 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 10618
  total_bytes: 10618
}

2022-10-11 03:45:30,561 INFO    Thread-8  :20914 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 10618
  total_bytes: 10618
}

2022-10-11 03:45:30,663 INFO    Thread-8  :20914 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 10618
  total_bytes: 10618
}

2022-10-11 03:45:30,765 INFO    Thread-8  :20914 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 10618
  total_bytes: 10618
}

2022-10-11 03:45:30,867 INFO    Thread-8  :20914 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 10618
  total_bytes: 10618
}

2022-10-11 03:45:31,274 INFO    Thread-8  :20914 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 10618
  total_bytes: 10618
}

2022-10-11 03:45:31,641 INFO    Thread-8  :20914 [wandb_run.py:_on_finish():1831] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 10618
  total_bytes: 10618
}
local_info {
}

2022-10-11 03:45:33,268 INFO    Thread-8  :20914 [wandb_run.py:_footer_history_summary_info():2865] rendering history
2022-10-11 03:45:33,269 INFO    Thread-8  :20914 [wandb_run.py:_footer_history_summary_info():2894] rendering summary
2022-10-11 03:45:33,271 INFO    Thread-8  :20914 [wandb_run.py:_footer_sync_info():2822] logging synced files
